src/dstore/pmix_esh.c

```diff
@@ -2126,15 +2128,15 @@ static size_t put_data_to_the_end(ns_track_elem_t *ns_info, seg_desc_t *dataseg,
     offset = global_offset % _data_segment_size;
 
     /* We should provide additional space at the end of segment to place EXTENSION_SLOT to have an ability to enlarge data for this rank.*/
-    if (sizeof(size_t) + ESH_KEY_SIZE(key, size) + EXT_SLOT_SIZE(key) > _data_segment_size) {
+    if ((sizeof(size_t) + ESH_KEY_SIZE(key, size) + EXT_SLOT_SIZE(key)) > _data_segment_size) {
```

Comment:
According to this:
http://en.cppreference.com/w/c/language/operator_precedence
this line was fine.
can you explain what two errors you identified?